plaintext/plaintext.go

```diff
@@ -0,0 +1,175 @@
+package plaintext
+
+import (
+    "bytes"
+    "io"
+    "strings"
+    "golang.org/x/net/html"
+    "github.com/microcosm-cc/bluemonday"
```

Comment:
I'm concerned about adding a new dependency to this SDK. Generally, we want to be as dependency free as possible. Maybe we release this functionality separately (or just provide the example code) and then demonstrate how it can be used in the USE_CASES.md section. So, if needed, a developer can easily add it to their project. 

We can then include a note in our main README that a best practice for sending emails is to include both HTML and text and we can point to this example as a solution in case they don't want to roll their own.

Reply:
Avoiding XSS requires parsing geared toward aggressive removal of undesired content which I hope the bluemonday library, being security conscious should do. No Sendgrid customer wants to be a messenger of a payload that comprimizes a users security or privacy so I thought it was worth including the dependency.

I've removed the dependency and added some rudimentary removal of <script> content. There are also 'javascript:' URLs to watch for and I don't know what else.

Perhaps the issue of XSS and security should be explored in a separate issue.